docs/source/installation.rst

```diff
@@ -2,6 +2,78 @@
 Installation
 ============
 
-At the command line::
+FURY supports Python 3.5+. You can currently still use Python 2.7 but it will be removed as the Python 2.7 end of life
+is on December 31st 2019.
+
+Dependencies
+------------
+
+The mandatory dependencies are:
+
+- numpy >= 1.7.1
+- vtk >= 8.1.0
+- scipy >= 0.9
+
+The optional dependencies are:
+
+- matplotlib >= 2.0.0
+- dipy >= 0.16.0
+
+
+Installation with PyPi
+----------------------
+
+Just use the `pip` command line::
```

Comment:
```suggestion
In a terminal, issue the following command::
```

Reply:
The word "just" is problematic: https://www.excitant.co.uk/just-is-a-dangerous-word/

Reply:
Ahaha, I read the opposite on a LinkedIn article 😄 . But I prefer your sentence. (I will find this article)